src/components/Missions.js

```diff
@@ -10,6 +12,10 @@ const Missions = () => {
     if (!missions.length) dispatch(fetchMissions());
   }, []);
 
```

Comment:
Good job configuring that you only fetch data once. I went to add this to my implementation, as I was sending requests each time the rockets page was loading.